cmd/sortfeatures/README.md

```diff
@@ -0,0 +1,84 @@
+# sortfeatures
+
+`sortfeatures` is a command-line tool that sorts feature declarations in Kubernetes feature files alphabetically.
+
+## Purpose
+
+In Kubernetes, feature gates should be listed in alphabetical, case-sensitive (upper before any lower case character)
+order to reduce the risk of code conflicts and improve readability. This tool enforces this convention by automatically
+sorting feature declarations in specified files.
+
+## Usage
+
+```bash
+sortfeatures [flags] [files...]
+```
+
+### Flags
+
+- `--files`: One or more file paths to process
+- `--force`, `-f`: Force update even if the file is already sorted
+
+### Examples
+
+Process specific files:
+```bash
+sortfeatures --files pkg/features/kube_features.go staging/src/k8s.io/apiserver/pkg/features/kube_features.go
```

Comment:
Isn't only `pkg/features/kube_features.go` a parameter of `--files` and the rest are positional arguments?

The `--files` parameter seems unnecessary. I would use only positional arguments.